lib/classes/class-settings.php

```diff
@@ -26,7 +26,7 @@
           'on_fly'                 => array('WP_STATELESS_MEDIA_ON_FLY', 'false'), 
           'bucket'                 => array('WP_STATELESS_MEDIA_BUCKET', ''), 
           'root_dir'               => array('WP_STATELESS_MEDIA_ROOT_DIR', ''), 
-          'key_json'               => array('WP_STATELESS_MEDIA_JSON_KEY', ''), 
```

Comment:
What is it's purpose? Why have `WP_STATELESS_MEDIA_JSON_KEY ` when `WP_STATELESS_MEDIA_JSON_KEY ` exists? Just curious.